src/themes/oneui/base-variables/_sizing.scss

```diff
@@ -18,6 +18,9 @@ $spacing-5x: 5 * $spacing-normal !default;
 
 $margin:    0.5 * $spacing-normal !default;
 
+// containers
+$site-container-size: 1024px !default;
```

Comment:
Can we set this to 1150 for starters? Most displays support at least 1280 nowadays. We can still fine-tune it later but this is what I currently use in my designs